controllers/provisioning_controller.go

```diff
@@ -250,6 +250,10 @@ func (r *ProvisioningReconciler) Reconcile(ctx context.Context, req ctrl.Request
 			"unable to put %q ClusterOperator in Available state", clusterOperatorName)
 	}
 
+	if _, ok := baremetalConfig.Annotations["provisioning.metal3.io/paused"]; ok {
+		return ctrl.Result{RequeueAfter: time.Minute}, fmt.Errorf("provisioning CR paused")
```

Comment:
nit: `fmt.Errorf` is an overkill, `errors.New` is enough

Reply:
Done